blueman/plugins/applet/StandardItems.py

```diff
@@ -24,53 +24,34 @@ class StandardItems(AppletPlugin):
     def on_load(self, applet):
         self.Applet = applet
 
-        applet.Plugins.Menu.Register(self, Gtk.SeparatorMenuItem(), 21)
+        applet.Plugins.Menu.add(self, 21)
 
-        self.new_dev = create_menuitem(_("_Setup New Device") + "...", get_icon("document-new", 16))
-        self.new_dev.connect("activate", self.on_setup_new)
+        self.new_dev = self.Applet.Plugins.Menu.add(self, 30, text=_("_Setup New Device") + "...",
+                                                    icon_name="document-new", callback=self.on_setup_new)
 
-        self.Applet.Plugins.Menu.Register(self, self.new_dev, 30)
+        self.Applet.Plugins.Menu.add(self, 31)
 
-        self.Applet.Plugins.Menu.Register(self, Gtk.SeparatorMenuItem(), 31)
+        self.send = self.Applet.Plugins.Menu.add(self, 40, text=_("Send _Files to Device") + "...",
+                                                 icon_name="blueman-send-file", callback=self.on_send)
 
-        self.send = create_menuitem(_("Send _Files to Device") + "...", get_icon("blueman-send-file", 16))
-        self.send.connect("activate", self.on_send)
+        self.Applet.Plugins.Menu.add(self, 51)
 
-        self.Applet.Plugins.Menu.Register(self, self.send, 40)
+        self.devices = self.Applet.Plugins.Menu.add(self, 60, text=_("_Devices") + "...", icon_name="blueman",
+                                                    callback=self.on_devices)
 
-        self.Applet.Plugins.Menu.Register(self, Gtk.SeparatorMenuItem(), 51)
+        self.adapters = self.Applet.Plugins.Menu.add(self, 70, text=_("Adap_ters") + "...", icon_name="blueman-device",
+                                                     callback=self.on_adapters)
 
-        self.devices = create_menuitem(_("_Devices") + "...", get_icon("blueman", 16))
-        self.devices.connect("activate", self.on_devices)
+        self.Applet.Plugins.Menu.add(self, 80, text=_("_Local Services") + "...", icon_name="preferences-desktop",
```

Comment:
We should probably use the Unicode ellipsis instead of three dots. Example ``…``. It was like that before but if you are changing things anyway 😄